web/src/lib/roadmap/buildUserRoadmap.test.ts

```diff
@@ -866,4 +866,29 @@ describe("buildUserRoadmap", () => {
       expect(getLastCalledWith(mockRoadmapBuilder)[0].addOns).toEqual([]);
     });
   });
+
+  describe("raffle-bingo games business", () => {
```

Comment:
I might just shorten this to reflect the title of the add-on.
```suggestion
  describe("raffle-bingo-games", () => {
```